package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "react-dev": "webpack --watch",
-    "server-dev": "nodemon server/index.js"
+    "server-dev": "nodemon server/index.js",
+    "seed": "node helpers/index.js"
```

Comment:
@gigicupp Looks good! I assume that you have verified that when you run this script, it indeed populates your database with all the information you need? You are so good at mySql, great job!

Reply:
@masoncpott yep